src/args.rs

```diff
@@ -731,8 +731,10 @@ impl<'a> ArgMatches<'a> {
         let mut specs = vec![
             "path:fg:magenta".parse().unwrap(),
             "line:fg:green".parse().unwrap(),
+            "column:fg:yellow".parse().unwrap(),
```

Comment:
More features means more code that I need to maintain. For example, if we ever added code to auto-detect the best color settings (which seems like something Windows users might appreciate), then the more color settings we have, the harder that code will be to write.

Features need to be decided upon carefully. Once they're added, we can essentially never remove them. Sorry.